odl/tomo/operators/ray_trafo.py

```diff
@@ -214,26 +211,54 @@ def geometry(self):
 
     def _call(self, x, out=None):
         """Forward project ``x`` and store the result in ``out`` if given."""
+        if self.domain.is_rn:
+            return self._call_real_fp(x, out)
+
+        elif self.domain.is_cn:
+            result_parts = []
+            for part in ['real', 'imag']:
+                x_part = getattr(x, part)
+                out_part = getattr(out, part, None)
+                result_parts.append(self._call_real_fp(x_part, out_part))
```

Comment:
Yeah, I've rewritten it without loop. The original was a mix of copy from the old implementation and some show-off with `getattr` Kung-Fu. Much simpler now.